test_pull_requests

```diff
@@ -359,6 +361,78 @@ def compose_bot_comment(prefix, comment_data={})
   updated_comment
 end
 
+class FlushableStore
+  def initialize(c_file=nil)
+    @cache_file = nil
+    @cache = {}
+    if c_file
+      @cache_file = File.expand_path(c_file)
+      if File.exists? @cache_file
```

Comment:
I don't want the script to die on load failure, and I don't want to swallow the errors either.